apps/RiiTag RPC/install-64

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+version=v1.3.2
+
+"{$DIRECTORY}/manage" install-if-not-installed ArmCord | error "Failed to install dependency: ArmCord"
```

Comment:
already told you to fix both of these things, stop closing them

`|` is a pipe. you wan tot use `||`

`{$DIRECTORY}` is not normal, use `${DIRECTORY}`

Reply:
i did not get what you were saying

Reply:
(before)